microsoftgraph/microsoft-graph-explorer-v4
# Task: Refactor permissions

## Overview
(Technical debt) Refactors Permission.tsx from a class to a functional component

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes
It is now much easier to read and make adjustments to

## Testing Instructions

Check out this branch locally
Ensure there is a query on the query textbox
Click on Modify Permissions. Notice that the permissions are still loaded
Sign in with any account
Click on the settings icon (...)
Click on 'Select permissions'
Notice that all permissions are loaded as expected
